crates/uv-client/src/base_client.rs

```diff
@@ -306,7 +321,11 @@ impl<'a> BaseClientBuilder<'a> {
             .expect("Failed to build HTTP client.")
     }
 
-    fn apply_middleware(&self, client: Client) -> ClientWithMiddleware {
+    fn apply_middleware(
+        &self,
+        client: Client,
+        url_auth_policies: Option<&UrlAuthPolicies>,
```

Comment:
Perhaps I'm missing something, but I'm confused by this signature change. Shouldn't this read `self.url_auth_policies` directly like all the other configuration is handled here?

Reply:
Yes, that was an oversight. Fixed.